.circleci/config.yml

```diff
@@ -5,11 +5,24 @@ jobs:
   build:
     docker:
       # Specify the Ruby version you desire here
-      - image: cimg/ruby:3.1-node
+      - image: cimg/ruby:3.1-browsers
 
     working_directory: ~/identity-dev-docs
     steps:
       - checkout
+      
+      - run:
+          name: Switch Node.js version
+          command: |
+            wget -qO- https://raw.githubusercontent.com/nvm-sh/nvm/v0.39.0/install.sh | bash
```

Comment:
out of curiosity, why `0.39.0` and not `0.39.7`? Is there a better way to install `^0.39`?

Reply:
hm, this version is from 10/2021

Reply:
That's a good call-out. This was [copied from `identity-site`](https://github.com/GSA-TTS/identity-site/blob/9a91f4fed83f8818843244913e4ea87e77af1efb/.circleci/config.yml#L11-L18), where it was the latest version at the [time of introduction](https://github.com/GSA-TTS/identity-site/pull/990). It was pinned to the specific version to avoid unexpected breakage from changing versions and for security, though in retrospect there's not much security assurances relying on git tags, since they can be deleted and recreated. SHA might be a better bet.

I'd think this could be bumped to the current latest version, assuming it works.

Reply:
I just grabbed the same version/command that the identity-design-system is using for the sake of consistency -- https://github.com/18F/identity-design-system/blob/948b3326379f182fd4d5dbabcc10daca510763d7/.circleci/config.yml#L13-L20